insights/tests/test_soscleaner.py

```diff
@@ -1,15 +1,77 @@
-from insights.contrib.soscleaner import SOSCleaner
+from collections import namedtuple
+from contextlib import contextmanager
+from json import dump
+from os import mkdir
+from os.path import join
+from shutil import rmtree
+from tempfile import mkdtemp
 
+from mock.mock import call
 from mock.mock import Mock
+from mock.mock import patch
 from pytest import mark
 
+from insights.client.config import InsightsConfig
+from insights.client.data_collector import CleanOptions
```

Comment:
I was hoping we could reduce this interdependency, but it appears that is going to require more work. For example, why is `CleanOptions`, a structure used by `SOSCleaner`, declared in `insights.client.data_collector`?

Reply:
I was surprised by that too. And why is it a structure anyway.